src/components/fields/text-field/text-field.tsx

```diff
@@ -137,6 +137,16 @@ export const TextField = (props: IGenericFieldProps<ITextFieldSchema | IEmailFie
 		setStateValue(event.target.value);
 	};
 
+	const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>): void => {
+		if (uiType === "numeric-field") {
+			// handle suppressing exponent input for numeric fields
+			const notAllowedKeys = ["e", "E", "+", "-"];
```

Comment:
`-` could be a valid input for negative numbers though. could we only prevent `e` and `E`?